### PR TITLE
More topics to start Python-related conversations with.

### PR DESCRIPTION
**Pull request by Xithrius**
Continuation of https://github.com/python-discord/sir-lancebot/pull/763.

### CODE REVIEW

File: bot/resources/evergreen/py_topics.yaml [hunk 1]
```diff
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language? If not, what is it?
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:27]:
```suggestion
    - Was Python your first programming language? If not, what was?
```

File: bot/resources/evergreen/py_topics.yaml [hunk 2]
```diff
@@ -23,6 +23,18 @@
...
+    - What's your favourite aspect of python development? (Backend, web dev, game dev, machine learning, ai, etc.)
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:28]:
```suggestion
    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
```

File: bot/resources/evergreen/py_topics.yaml [hunk 3]
```diff
@@ -23,6 +23,18 @@
...
+    - In what ways has python discord helped you with python?
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:29]:
```suggestion
    - In what ways has Python Discord helped you with Python?
```
And other cases of `python` -> `Python`

File: bot/resources/evergreen/py_topics.yaml [hunk 4]
```diff
@@ -23,6 +23,18 @@
...
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What is it and why?
+    - How often do you program in python?
+    - How would you learn a new library if needed to do so?
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:34]:
```suggestion
    - How would you approach learning a new Python library?
```
Maybe, previous was fine though as well

File: bot/resources/evergreen/py_topics.yaml [hunk 5]
```diff
@@ -23,6 +23,18 @@
...
+    - Have you ever worked with a microcontroller or anything physical with python before?
+    - How good would you say you are at python so far, beginner/intermediate or advanced?
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:36]:
```suggestion
    - How good would you say you are at Python so far: beginner, intermediate or advanced?
```

File: bot/resources/evergreen/py_topics.yaml [hunk 6]
```diff
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Is python your first programming language? If not, what is it?
+    - What's your favourite aspect of python development? (Backend, web dev, game dev, machine learning, ai, etc.)
+    - In what ways has python discord helped you with python?
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What is it and why?
```
Comment (wookie184) [inline bot/resources/evergreen/py_topics.yaml:32]:
```suggestion
    - Have you ever been unable to finish a python project? What was it and why?
```

**Review by wookie184 — CHANGES_REQUESTED** [review]
Just grammar and wording. There are a couple of `python`s that should be `Python`s I didn't comment on. Could you also fix capitalization on line 19, as `PyPi`, should be `PyPI`. Not part of this PR but probably not worth it's own one ig.

**Comment by Xithrius** [comment]
The syntax has changed for the yaml file, given the `:` near the end of it. I think I need to escape that with one singular `\`.

**Comment by wookie184** [comment]
Oh interesting, that makes sense.

Also still some `python`s that need capitalizing btw

**Comment by NIRDERIi** [comment]
I wish I could help with the Python caps, but it'll be a lot of trouble to start a new fork for it

**Comment by NIRDERIi** [comment]
**Caps issue**:
Line: `26`. Word: `python` -> `Python`. Reason: `Name`
Line `31`. Word: `python` -> `Python` Reason: `Name`
Line `32`. Word: `python` -> `Python` Reason: `Name`
Line `33`. Word: `python` -> `Python` Reason: `Name`
Line `35`. Word: `python` -> `Python` Reason: `Name`
Line `67`. Word: `what` -> `What`. Reason: `After question mark (?)`

**Review by brad90four — COMMENTED** [review]
Is there a consistent rule for capitalized or un-capitalized "python"? 
Lines 26, 31, 32, 33, 35 are un-capitalized, but the others are capitalized. 

For line 36, The wording seems a bit clunky. Maybe "What Python level/skill/knowledge do you think you are at: Beginner, Intermediate or Advanced?"

(Just chiming in, wanting to get started with contributing!)

**Comment by NIRDERIi** [comment]
Wdym? They are not capitalized which is incorrect spelling, the others are capitalized because it is the correct spelling because it is a name. Unless I misunderstood you?

File: bot/resources/evergreen/py_topics.yaml [hunk 7]
```diff
@@ -23,6 +23,18 @@
...
+    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
+    - In what ways has Python Discord helped you with Python?
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What was it and why?
+    - How often do you program in python?
+    - How would you approach learning a new Python library?
+    - Have you ever worked with a microcontroller or anything physical with python before?
+    - How good would you say you are at Python so far: beginner, intermediate or advanced?
+    - Have you ever tried making your own programming language?
```
Comment (brad90four) [inline bot/resources/evergreen/py_topics.yaml:37]: - What do you plan on eventually achieving with Python?
    - Was Python your first programming language? If not, what was?
    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
    - In what ways has Python Discord helped you with Python?
    - Are you currently using Python professionally, for education, or as a hobby?
    - What is your process when you decide to start a project in Python?
    - Have you ever been unable to finish a Python project? What was it and why?
    - How often do you program in Python?
    - How would you approach learning a new Python library?
    - Have you ever worked with a microcontroller or anything physical with Python before?
    - What Python level do you think you are at: Beginner, Intermediate or Advanced?
    - Have you ever tried making your own programming language?

**Comment by brad90four** [comment]
> 
> 
> Wdym? They are not capitalized which is incorrect spelling, the others are capitalized because it is the correct spelling because it is a name. Unless I misunderstood you?

I was just unsure if there was some un-written rule of when to use "python" vs "Python". For example I could say I have a python script written in Python. When it is an adjective lower case seems fine, but as a noun it seems more proper to use the capitalization.

File: bot/resources/evergreen/py_topics.yaml [hunk 8]
```diff
@@ -23,6 +23,18 @@
...
+    - Was Python your first programming language? If not, what was?
+    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
+    - In what ways has Python Discord helped you with Python?
+    - Are you currently using Python professionally, for education, or as a hobby?
+    - What is your process when you decide to start a project in python?
+    - Have you ever been unable to finish a python project? What was it and why?
+    - How often do you program in python?
+    - How would you approach learning a new Python library?
+    - Have you ever worked with a microcontroller or anything physical with python before?
+    - How good would you say you are at Python so far: beginner, intermediate or advanced?
```
Comment (Shivansh-007) [inline bot/resources/evergreen/py_topics.yaml:36]: This is loaded incorrectly (not according to what we want it to be) as `:` syntax makes it a dictionary.

Screenshot from 2021-08-08 17-13-24

```suggestion
    - "How good would you say you are at Python so far: beginner, intermediate or advanced?"
```

File: bot/resources/evergreen/py_topics.yaml [hunk 9]
```diff
@@ -23,6 +23,18 @@
     - When you were first learning, what is a resource you wish you had?
     - What is something you know now, that you wish you knew when starting out?
     - What is something simple that you still error on today?
+    - What do you plan on eventually achieving with python?
+    - Was Python your first programming language? If not, what was?
+    - What's your favourite aspect of Python development (e.g. backend, web dev, game dev, machine learning, ai)?
```
Comment (Shivansh-007) [inline bot/resources/evergreen/py_topics.yaml:28]: Isn't `web dev` included with `backend`? You cannot do frontend with it...

Comment (Xithrius) [reply bot/resources/evergreen/py_topics.yaml:28]: Technically `web dev` could just be `frontend`, but for clarity I think I'll just replace `web dev` with `frontend`

**Comment by Xithrius** [comment]
@NIRDERIi @Shivansh-007 fixed up.

**Comment by NIRDERIi** [comment]
Looks great for me!

**Review by Shivansh-007 — APPROVED** [review]

**Review by brad90four — APPROVED** [review]

**Review by dawnofmidnight — APPROVED** [review]
:shipit:

**Review by ChrisLovering — APPROVED** [review]

**Pull request merged**